yarn.lock

```diff
@@ -20125,25 +20125,20 @@ minimist-options@^3.0.1:
     arrify "^1.0.1"
     is-plain-obj "^1.1.0"
 
-minimist@0.0.5:
-  version "0.0.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.5.tgz#d7aa327bcecf518f9106ac6b8f003fa3bcea8566"
-  integrity sha1-16oye87PUY+RBqxrjwA/o7zqhWY=
+minimist@0.0.5, minimist@^0.2.1:
+  version "0.2.1"
+  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.2.1.tgz#827ba4e7593464e7c221e8c5bed930904ee2c455"
+  integrity sha512-GY8fANSrTMfBVfInqJAY41QkOM+upUTytK1jZ0c8+3HdHrJxBJ3rF5i9moClXTE8uUSnUo8cAsCoxDXvSY4DHg==
 
 minimist@0.0.8:
   version "0.0.8"
```

Comment:
Looks like there's still a transitive dependency that's resolving to `0.0.8`?

Reply:
This should have been fixed by upgrading `mapbox-gl` to v1.9.0 which I did in #61135, but I'm not sure why `mapbox-gl` still retains a reference to `minimist@0.0.8`. I need to rebase with master anyway since I've recently landed a few things related to this there, so hopefully this will be fixed once that's done

Reply:
Hmm for some reason even though I have upgraded to `mapbox-gl@1.9.0`, it still retains a reference to the old version of `minimist`. Will investigate why...

Reply:
https://github.com/mapbox/mapbox-gl-js/pull/9425#issuecomment-605414888

Reply:
For now I added a forced upgrade of `mapbox-gl/minimist`

Reply:
However, there are still two references to `minimist@0.0.8` left, which are coming from `mkdirp`. Though I've already tried to force-upgrade those. Why yarn doesn't follow those instructions is a mystery.

Reply:
Fixed